src/AFQMC/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ IF(ENABLE_CUDA)
   target_link_libraries( afqmc ${CUDA_curand_LIBRARY} )
 ELSE(ENABLE_CUDA)
   ADD_LIBRARY(afqmc ${AFQMC_SRCS})
+  TARGET_LINK_LIBRARIES(afqmc libgfortran.a)
```

Comment:
checked in by mistake?

Reply:
Yes but I do need this to build with afqmc is on.  Separate issue.

Reply:
This will break the build on systems without gfortran. Can not merge this line.